pkg/virt-v2v/customize/scripts/rhel/run/test-network_config_util.sh

```diff
@@ -23,7 +23,7 @@ test_dir() {
     export IFQUERY_CMD="
       podman run
       -v $TEST_DIR/etc/network:/etc/network
-      cmd.cat/ifquery ifquery
+      quay.io/kubev2v/ifquery:latest ifquery
```

Comment:
Note: the ifquery stopped working so I have uploaded the image to the kubev2v so we can use it in our CI